roles/openshift_prometheus/tasks/uninstall_prometheus.yaml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: delete router-metrics cluster role
+  oc_obj:
+    state: absent
+    kind: clusterrole
+    name: router-metrics
```

Comment:
Doesn't the cluster-role get removed when the namespace is deleted?

Reply:
I had the feeling that it wasn't but I'll try again.

Reply:
In that case shouldn't we also delete `cluster-reader` here?

Reply:
If I'm not wrong, `cluster-reader` is a built-in cluster role so it shouldn't be deleted.